pkg/annotation/annotation.go

```diff
@@ -6,4 +6,7 @@ const (
 
 	// UpgradingToNodePools is set to True during the first cluster upgrade to node pools release.
 	UpgradingToNodePools = "release.giantswarm.io/upgrading-to-node-pools"
+
+	// NodeTerminateUnhealthy the name of the annotation needed in the Cluster CR to enable the terminate-unhealthy feature.
+	NodeTerminateUnhealthy = "alpha.node.giantswarm.io/terminate-unhealthy"
```

Comment:
you can import that from apiextensions annotation package

Reply:
thanks